### ISSUE TITLE
[Discussion] capturetheflag wiki

### ISSUE THREAD

**Issue by ClobberXD** [rubenwardy/capturetheflag#158]
IMHO, this wiki isn't the right place for the wikipedia definition of "Capture The Flag", and a Building Bases 101. While being very useful and relevant, I feel that they might actually be of help to someone if they're posted on the forums; I don't see anyone paying a visit to the (sub-)game's GitHub repo to learn how to build bases. Instead, the wiki should contain contribution guide-lines, the internal architecture of the (sub-)game, list of mods with a short description, and other such technical content.

Maybe @rubenwardy could create a dedicated wiki at ctf.rubenwardy.com/wiki, for the non-technical aspects of the game-play. This wiki could have pages for bases, teams, strategy, weapons, team-work, maps, map-creation 101, and the like.

This is a discussion about the wiki content, and I've put across my thoughts about the same. Please share your thoughts, opinions and ideas below :)

Pinging @afshaan4 and @farooqkz

**Comment by afshaan4** [comment]
yeah, i just saw it and added a bit of stuff, didn't realise its pointless here

**Comment by farooqkz** [comment]
My plan for the home page of wiki is a short introduction to CTF, another short introduction to rubenwardy's CTF and then links to a few useful pages.
Wiki could contain both technical and non-technical stuff. This depends on yourselves and https://help.github.com/articles/about-github-wikis/ hasn't said whether it should be technical or non-technical. Also there are many projects which use their wiki for non-technical stuff. like https://github.com/SuperTux/supertux/wiki.
rubenwardy surely could create another wiki, He could fire up two wikis for technical and non-technical stuff. But he will have to spend time configuring and maintaining, spend money buying a server for it and other costs. Does it worth it? Github is already providing something for free and you just have to use it.
At last no one's paying attention to this probably because no one knows something's here.

**Comment by ClobberXD** [comment]
@farooqkz I feel that your efforts would be wasted since people come to _this_ repo to contribute / mess around with the code. Here's what GitHub's Help says about Wikis (it's from the page you linked):

> GitHub Wikis are a place in your repository where you can share long-form content about your project, _**such as how to use it, how it's been designed, manifestos on its core principles, and so on**_. Whereas a README is intended to quickly orient readers as to what your project can do, wikis can be used to provide additional documentation.

**TL;DR**: The wiki is a place for extensive documentation of the project itself.

> Also there are many projects which use their wiki for non-technical stuff. like https://github.com/SuperTux/supertux/wiki.

While I can't comment on the decision of SuperTux devs to use the GitHub wiki for both technical and non-technical content, it's their repo and their wish as to what content they add and how they maintain their wiki.

> But he will have to spend time configuring and maintaining, spend money buying a server for it and other costs.

That was just a casual suggestion, by the way.

But have you been to https://rubenwardy.com or https://ctf.rubenwardy.com? He already has his own domain, as well as a VPS which he uses for running the CTF server in addition to other things. Lots of wiki software available for free, which he can use (if he wants to)...

> At last no one's paying attention to this probably because no one knows something's here.

That's literally the point of this whole discussion.

**Comment by farooqkz** [comment]
This is a game, its usage for end users is playing. So we can have articles about playing it.
And about the costs, you should pay for resources you use whether you already have bought it or not.
Also let's don't forget costs of maintaining a wiki. different softwares have different costs, many are more simple and easy to maintain while you should spend more time on many others, but at all they won't maintain themselves.

**Comment by rubenwardy** [comment]
I think it's fine to use the wiki here (mostly because I can't be bothered to install wiki software on my server)

**Comment by AKryukov92** [comment]
Other options:
* Use any open wiki platform. For example wikia.com. It requires registration and is cluttered with advertisment.
* Use github pages (according to https://help.github.com/articles/user-organization-and-project-pages/ you just can write Markdown pages in a folder named "docs" located on the master branch)
* Start dedicated wiki repository, so Rubenwardy could just clone it and deploy on his server (there are a lot of potential technical issues anyway)

**Comment by farooqkz** [comment]
The first and last ideas don't seem fine. The last because if its costs and the first because we have a wiki platform here. But about the middle, @rubenwardy could create a new organization so capturetheflag could have an independent website for itself.
